client/state/initial-state.js

```diff
@@ -55,7 +72,7 @@ function loadInitialStateFailed( error ) {
 
 function persistOnChange( reduxStore ) {
 	reduxStore.subscribe( function() {
-		localforage.setItem( 'redux-state', serialize( reduxStore.getState() ) )
+		localforage.setItem( 'redux-state', addTimestamp( serialize( reduxStore.getState() ) ) )
```

Comment:
The `addTimestamp`/`removeTimestamp` calls could maybe be moved inside the `serialize`/`deserialize` functions, making this line a little simpler?